---
title: C6322 | Microsoft Docs
ms.date: 11/15/2016
ms.prod: visual-studio-dev14
ms.technology: vs-ide-code-analysis
ms.topic: reference
f1_keywords:
- C6322
helpviewer_keywords:
- C6322
ms.assetid: fb23d2b1-b2a0-465c-8bf5-ec039c6c7757
caps.latest.revision: 15
author: corob-msft
ms.author: corob
manager: jillfra
ms.openlocfilehash: 15b6aab53a5c964b95b44dfbc21379369effea4c
ms.sourcegitcommit: 6cfffa72af599a9d667249caaaa411bb28ea69fd
ms.translationtype: MT
ms.contentlocale: tr-TR
ms.lasthandoff: 09/02/2020
ms.locfileid: "77273343"
---
# <a name="c6322"></a>C6322
[!INCLUDE[vs2017banner](../includes/vs2017banner.md)]

Uyarı C6322: boş _except bloğu  
  
 Bu ileti _except bloğunda kod olmadığını gösterir. Sonuç olarak, özel durumlar işlenmemiş duruma gelebilir.  
  
## <a name="example"></a>Örnek  
 Aşağıdaki kod bu uyarıyı oluşturur:  
  
```  
#include <stdio.h>  
#include <excpt.h>  
#include <windows.h>  
  
void fd(char *pch)  
{  
   __try  
     {  
       // exception rasied if pch is null  
       *pch= 0 ;  
     }  
   __except(GetExceptionCode() == EXCEPTION_ACCESS_VIOLATION)  
     {  
     }  
}  
```  
  
 Bu uyarıyı düzeltmek için aşağıdaki kodu kullanın:  
  
```  
#include <stdio.h>  
#include <excpt.h>  
#include <windows.h>  
  
void f(char *pch)  
{  
   __try  
     {  
       // exception rasied if pch is null  
      *pch= 0 ;  
     }  
   __except(GetExceptionCode() == EXCEPTION_ACCESS_VIOLATION ?   
               EXCEPTION_EXECUTE_HANDLER : EXCEPTION_CONTINUE_SEARCH)  
     {  
       // code to handle exception  
       puts("Exception Occurred");     
     }  
}    
```  
  
## <a name="see-also"></a>Ayrıca Bkz.  
 [try-except ekstresi](https://msdn.microsoft.com/library/30d60071-ea49-4bfb-a8e6-7a420de66381)
